examples/testresample.rs

```diff
@@ -38,7 +41,7 @@ fn main() {
             off -= INBLOCK;
         }
 
-        println!("{:#?}", &fout[..out_len as usize]);
+        eprintln!("{:#?}", &fout[..out_len as usize]);
```

Comment:
The output should remain to `stdout`.

Reply:
At a first glance we (maybe wrongly) thought the output was not essential to this example, so we diverted to stderr (so it doesn't pollute the test output - see pull request #1).

Do you think we should rever this change? a better strategy to handle this wall of `println`s?

Reply:
The example code works this way:
- Generates some samples
- Output the resampled samples

You can compare the output with the similar C example and be sure at least that the rust wrapper (soon also a rust port of this resampling logic) behave as the C equivalent.